docs/multiplatform.md

```diff
@@ -64,17 +64,22 @@ Add the compiler dependencies in a top level `dependencies` block
 ```kotlin
 dependencies {
     // 1. Configure code generation into the common source set
-    kspCommonMainMetadata(libs.kotlinInject)
+    kspCommonMainMetadata(libs.kotlin.inject.compiler)
 
     // 2. Configure code generation into each KMP target source set
-    kspAndroid("me.tatarka.inject:kotlin-inject-compiler-ksp:0.7.2")
-    kspIosX64("me.tatarka.inject:kotlin-inject-compiler-ksp:0.7.2")
-    kspIosArm64("me.tatarka.inject:kotlin-inject-compiler-ksp:0.7.2")
-    kspIosSimulatorArm64("me.tatarka.inject:kotlin-inject-compiler-ksp:0.7.2")
+    listOf(
+        "kspAndroid",
+        "kspIosX64",
+        "kspIosArm64",
+        "kspIosSimulatorArm64",
+        // add more targets here...
```

Comment:
This comment could stay though (IMO not worth listing out all of the supported targets here).